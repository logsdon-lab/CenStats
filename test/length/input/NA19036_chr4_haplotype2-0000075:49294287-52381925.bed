version https://git-lfs.github.com/spec/v1
oid sha256:2ba2fb0646588bcc7515f16ea846c8f3202b641fcd93fa7ca3fafd8dd1b147a6
size 73158
