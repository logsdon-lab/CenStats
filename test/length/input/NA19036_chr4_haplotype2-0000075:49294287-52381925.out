version https://git-lfs.github.com/spec/v1
oid sha256:af453e61e662ac4aa02558820ebc6fcec5a9ded78b166ff5da51d1cb463b1d5c
size 196230
